### PR TITLE
Update version to 0.7.0.dev1 and update docs 0.6.3 -> 0.6.4

### PR DESCRIPTION
**Pull request by pcmoritz**

### CODE REVIEW

File: python/ray/__init__.py [hunk 1]
```diff
@@ -95,7 +95,7 @@
 from ray.runtime_context import _get_runtime_context  # noqa: E402
 
 # Ray version string.
-__version__ = "0.6.4"
+__version__ = "0.7.0.dev0"
```
Comment (robertnishihara) [inline python/ray/__init__.py]: Maybe `.dev1`?

Comment (pcmoritz) [reply python/ray/__init__.py]: fine with me

**Review by robertnishihara — APPROVED** [review]

**Comment by robertnishihara** [comment]
@pcmoritz can you also submit a PR updating the docs and the autoscaler configs and whatever else to use `0.7.0.dev1`.

**Comment by pcmoritz** [comment]
Yeah!

**Comment by pcmoritz** [comment]
https://github.com/ray-project/ray/pull/4282

**Pull request merged**